scripts/test-ssr/bin/test-ssr.js

```diff
@@ -1,4 +1,8 @@
 #!/usr/bin/env node
 
-require('@fluentui/scripts-ts-node/register');
```

Comment:
this nukes test-ssr bin with latest swc https://github.com/TypeStrong/ts-node/pull/2062